src/js/components/explorer/detail/DetailContent.jsx

```diff
@@ -205,16 +208,25 @@ export default class DetailContent extends React.Component {
                 total={this.props.total}
                 data={this.props.data}
                 goDeeper={this.props.goDeeper}
+                lastUpdate={this.props.lastUpdate}
```

Comment:
The `lastUpdate` prop is being passed down to the `ExplorerTreemap` and `ExplorerTable` components, where it is used as a parameter in the `goToUnreported` function. That function is defined in `DetailContentContainer`, where the same `lastUpdate` that is being passed down is already available in the state.